src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -49,10 +49,16 @@
  */
 public final class ColonyManager
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 35 to the maximum authorized 30 or less. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "ColonyManager" has 52 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)